djstripe/models.py

```diff
@@ -459,7 +459,7 @@ def sync_current_subscription(self, cu=None):
         if sub:
             try:
                 sub_obj = self.current_subscription
-                sub_obj.plan = djstripe_settings.plan_from_stripe_id(sub.plan.id)
+                sub_obj.plan = sub.plan.id
```

Comment:
Instead of just saving the ID, current subscription should have a foreign key to Plan.